packages/modul-components/src/components/file-upload/file-upload.lang.fr.json

```diff
@@ -7,8 +7,8 @@
         "dropFile.p": "Déposer les fichiers ici ou",
         "selectFile": "Sélectionner un fichier",
         "selectFile.p": "Sélectionner des fichiers",
-        "format": "Format accepté",
-        "format.p": "Formats acceptés",
+        "format": "Format accepté%(_NBSP_)s",
+        "format.p": "Formats acceptés%(_NBSP_)s",
         "import": "En cours d'importation",
```

Comment:
Salut @jipigi 🥳

Reply:
On aurait pu mettre l'espace insécable directement aussi dans le html

Reply:
Salut  @chuckmah, it's been a while ! J'ai vérifié avec Audrey et on a pas d'espace en anglais, je me suis dit que sa place était dans le fichier de langue ;)